### PR TITLE
Add test to verify ISL cost is NOT increased due to connection loss

### PR DESCRIPTION
**Pull request by ylobankov**
Closes: #1316.

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/IslCostSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        def switches = topology.getActiveSwitches()
+        List<List<PathNode>> allPaths = []
+        List<PathNode> aswitchPath
+
+        [switches, switches].combinations().findAll { src, dst -> src != dst }.find { Switch src, Switch dst ->
+            allPaths = topologyEngineService.getPaths(src.dpId, dst.dpId)*.path
+            aswitchPath = allPaths.find { pathHelper.getInvolvedIsls(it).find { it.aswitch } }
+            aswitchPath
+        } ?: assumeTrue("Wasn't able to find suitable switch pair", false)
+
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/IslCostSpec.groovy]: Not clear why are you performing a lot of strange actions above. Why do you need paths between switches at all if you are not creating any flows. If you need an aswitch isl to break just do `def aswitchIsl = topology.islsForActiveSwitches.find { it.aswitch }`

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/IslCostSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        when: "Remove a-switch rules to break link between switches"
+        int islCost = pathHelper.getCost(aswitchPath)
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/IslCostSpec.groovy]: you are naming it 'islCost', but taking the cost of the whole path. should be `int islCost = pathHelper.getCost(aswitchIsl)`

Comment (rtretyak) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/IslCostSpec.groovy]: by the way, it is probably makes sense to move getCost(Isl isl) method to islUtils intead of pathHelper.

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/SwitchFailuresSpec.groovy [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package org.openkilda.functionaltests.spec
+package org.openkilda.functionaltests.spec.isl
```
Comment (rtretyak) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/SwitchFailuresSpec.groovy]: we already have 'spec.links' package. I find it pretty much the same meaning as 'isl'. Can we leave either isls or links?

Comment (ylobankov) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/SwitchFailuresSpec.groovy]: Let's have a separate discussion about this offline.

**Pull request merged**